src/test/java/io/projectriff/invoker/GrpcIsolatedTests.java

```diff
@@ -159,7 +168,7 @@ public void appClassPath() throws Exception {
 		assertThat(result).contains("10");
 	}
 
-	@Test(expected = IllegalStateException.class)
+	@Test(expected = Exception.class)
```

Comment:
What happened to the `IllegalStateException` that was previously expected?

Reply:
It morphed into a BeanCreationException and it comes from a different class loader, so you can't assert it in this way. I'd call that collateral damage for this change. Happy to fix it later, but I don't want to hold this PR up.

Reply:
Ok, it would be good to raise a bug to cover that fix.

Reply:
https://github.com/projectriff/java-function-invoker/issues/56

Reply:
https://github.com/projectriff/java-function-invoker/issues/56